### ISSUE TITLE
Ritual anchors not applying player loot effects

### ISSUE THREAD

**Issue by JayaramV-Hexaware** [HellFirePvP/AstralSorcery#577]
I'm playing skyblock 3 departed modpack.   I'm wearing a helm that gives me luck that makes mob kills drop a LOT of reliquary loot (over half a stack per mob).

The discidia ritual applies my luck to mob kills if I'm nearby.  However, the ritual anchor doesn't seem to do the same.  Don't know if it's intended, but it certainly isn't consistent behavior.

**Comment by Doomgull** [comment]
Considering the ritual is the source, not the anchor, it's pretty consistent. The only point of the anchor is to aim the ritual, nothing else is controlled by it.

**Comment by JayaramV-Hexaware** [comment]
Given that the purpose of an anchor is to focus the ritual effect elsewhere, it doesn't make too much sense to still consider the ritual to be the source.  Think about it; I'd keep the ritual in some place and redirect it's effect to my mob farm elsewhere for aesthetics.  Considering the ritual to still be the source diminishes the use of the anchor and forces me to rebuild the entire ritual near my mob farm to continue working as expected and hoped.
________________________________

**Comment by JayaramV-Hexaware** [comment]
After testing further, I can confirm that this is definitely a bug.  I placed the anchor right next to the ritual this time.  When the anchor is active, the luck isn't in effect and mobs drop normal loot.  When I break the anchor and let the ritual do its thing, the luck factor definitely comes into play.  I tested with over 100 mobs just to be sure and the anchor definitely disables the ritual's player luck from coming into effect - even if the anchor (and the mob) is right next to the ritual.
